sbt_for_bazel.bzl

```diff
@@ -0,0 +1,75 @@
+def _scala_major(ver):
+  parts = ver.split(".")
+  return "%s.%s" % (parts[0], parts[1])
+
+def _clean(target_name):
+  return target_name.replace("-", "_")
+
+def _jar_path(target, version, scala):
+  return "{target}/target/scala-{scala_version}/{target}_{scala_version}-{version}.jar".format(
+      target=target,
+      scala_version=_scala_major(scala),
+      version=version)
+
+
+def _sbt_jar(ctx):
+  ctx.download_and_extract(
+      ctx.attr.sbt_url,
+      ctx.path("sbt"),
+      ctx.attr.sbt_sha256,
+      '',
+      '')
+  targets = " ".join([ "%s/package" % t for t in ctx.attr.targets ])
```

Comment:
One thing I worry about is that the target that SBT uses and the directory it lives in may be different (eg Diorama uses camelCase internally, but dashes for directory names). I'm happy decreeing that the directory has to match the target name, but just want to check if perhaps there is an easy and robust way to handle the difference.